src/app/(simulation)/(large-layout)/amis/resultats/_components/GroupResults.tsx

```diff
@@ -9,16 +9,20 @@ import { useGetGroupStats } from '@/hooks/groups/useGetGroupStats'
 import { useIsGroupOwner } from '@/hooks/groups/useIsGroupOwner'
 import { useUser } from '@/publicodes-state'
 import { Group, Results } from '@/types/groups'
+import { QueryObserverResult } from '@tanstack/react-query'
 import Classement from './groupResults/Classement'
 import InviteBlock from './groupResults/InviteBlock'
 import OwnerAdminSection from './groupResults/OwnerAdminSection'
 import ParticipantAdminSection from './groupResults/ParticipantAdminSection'
 import PointsFortsFaibles from './groupResults/PointsFortsFaibles'
 
-type Props = {
+export default function GroupResults({
+  group,
+  refetchGroup,
+}: {
   group: Group
-}
-export default function GroupResults({ group }: Props) {
+  refetchGroup: () => Promise<QueryObserverResult<Group, Error>>
```

Comment:
Too bad remove participant is not PUT group... But anyway, that's alright

Reply:
Hm yes we need to be more cautious about using the right request type. I tend to use POST in 80% of cases